docs/compliance/index.md

```diff
@@ -0,0 +1,386 @@
+
+> [EDIT: INSERT "AZURE CYBER BLUEPRINT" PROGRAM OVERVIEW]
+
+## Overview
+
+The Automated Foundational Architecture for PCI DSS-compliant Environments (v1) provides guidance for the deployment of a PCI DSS-compliant Platform-as-a-Service (PaaS) environment suitable for handling sensitive payment card data. It showcases a common reference architecture and is designed to simplify adoption of Microsoft Azure. This foundational architecture (developed jointly with [Avyan Consulting](http://www.avyanconsulting.com/azureservices), a Microsoft MVP partner) illustrates an end-to-end solution to meet the needs of organizations seeking a cloud-based approach to reducing the burden and cost of deployment.
+
+This foundational architecture meets the requirements of stringent Payment Card Industry Data Security Standards (PCI DSS 3.2) for the collection, storage, and retrieval of payment card data. It demonstrates the proper handling of credit card data (including card number, expiration, and verification data) in a secure, compliant multi-tier environment deployed as an end-to-end Azure-based solution.
+
+This architecture is intended to serve as a foundation for customers to adjust to their specific requirements, and should not be used as-is in a production environment. Deploying an application into this environment without modification is not sufficient to completely meet the requirements of a PCI DSS-compliant solution. Please note the following:
+- This foundational architecture provides a baseline to help customers use Microsoft Azure in a PCI DSS-compliant manner.
+- Achieving PCI DSS-compliance requires that an accredited Qualified Security Assessor (QSA) certify a production customer solution.
+- Customers are responsible for conducting appropriate security and compliance reviews of any solution built using this foundational architecture, as requirements may vary based on the specifics of each customer’s implementation and geography.  
+
+## Architectural Diagram
+
+![](images/pci-architectural-diagram.png)
+
+## Solution Components
+
+The foundational architecuture is comprised of the following components:
+
+- **Architectural diagram**. The diagram shows the reference architecture used for the Contoso Webstore solution.
+- **Deployment templates**. In this deployment, [Azure Resource Manager templates](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-overview#template-deployment) are used to automatically deploy the components of the architecture into Microsoft Azure by specifying configuration parameters during setup.
+- **Automated deployment scripts**. These scripts help deploy the end-to-end solution. The scripts consist of:
+    - A module installation and [global administrator](https://docs.microsoft.com/en-us/azure/active-directory/active-directory-assign-admin-roles-azure-portal) setup script is used to install and verify that required PowerShell modules and global administrator roles are configured correctly.
```

Comment:
I don't think we need these details here. Automated deployment scripts is sufficient.